modules/ocf_kubernetes/templates/master/loadbalancer/haproxy.cfg.erb

```diff
@@ -67,6 +77,8 @@ backend kubernetes
   http-request add-header X-Forwarded-Port %[dst_port]
   mode http
   option forwardfor
+  option httpchk OPTIONS / HTTP/1.0
```

Comment:
per discussion in #802, let's GET /healthz, which should give 200. Also, imo lets just leave off the HTTP version. 1.0 is the default and if that ever updates our config won't be out of date.